test/e2eTests/i18n.js

```diff
@@ -29,24 +29,31 @@ I18n.prototype.loadTranslations = function () {
   var llTranslations = {};
   var locale;
   var dirs = fs.readdirSync(this.options.directory);
-  var obj = {};
   for (var i in dirs) {
     var name = this.options.directory + dirs[i];
 
     if (fs.statSync(name).isDirectory()) {
       var files = fs.readdirSync(name);
-      files.forEach(function (file) {
+      files.forEach((file) => {
 
         try {
           locale = path.basename(file).split('.').shift();
           if (locale.includes("pt")) {
-            obj = Object.assign(obj, JSON.parse(fs.readFileSync(self.getDirectoryPath(dirs[i], locale))));
```

Comment:
@sacuribraimo sorry I missed this in my initial review. This is easier to read when broken up. There's lots of folks who believe less lines of code is a good thing. I think readability is most important, so if I have to spend a few more lines of code to make things easier to understand I usually do it.

Reply:
The i18n needs some refactoring. Thanks for that.